### PR TITLE
Simplification, cleanup, SIS 2.0

### PR DESCRIPTION
**Pull request by pelaprat**
Depends on https://github.com/unizin/udp-schema/pull/27

This removes a lot of unused, old cruft. You only need now to run the UDP schema locally and run `gen_docs` and `gen_ls` for all of the documentation to be generated.

### CODE REVIEW

**Comment by pelaprat** [comment]
Added the SIS 2.0 loading schema.

File: udp/context-data/src/sis-2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,1047 @@
...
+      - code: DeclarationDate
+        description: The date that the student declared their intentions known regarding their Academic major specialization.
+        name: Declaration date
+      - code: MatriculationDate
+        description: The date that the student matriculated in their Academic major specialization.
+        name: Matriculation date
+      - code: WithdrawalDate
+        description: The date that the student withdrew their intentions to complete in the Academic major specialization.
+        name: Withdrawal date
+    name: PersonAcademicMajorSpecialization
```
Comment (karaboldt) [inline udp/context-data/src/sis-2.0.yaml]: Entities are spaced, not camel case (e.g. `Person academic major specialization`).

File: udp/context-data/src/sis-2.0.yaml [hunk 2]
```diff
@@ -0,0 +1,1047 @@
...
+        description: The date that the student withdrew their intentions to complete in the Academic major.
+        name: Withdrawal date
+    name: PersonAcademicMajor
+    system_of_record: true
+  - description: 'A student’s major specializations, defined by the transactions where they declared, withdrew, or matriculated their major specialization.'
+    elements:
+      - code: PersonId
+        description: The foreign key to the Person to which this relationship belongs.
+        fk: Person.SisIntId
+        name: SIS Internal Person ID
```
Comment (karaboldt) [inline udp/context-data/src/sis-2.0.yaml]: Wouldn't this element be required?

File: udp/context-data/src/sis-2.0.yaml [hunk 3]
```diff
@@ -0,0 +1,1047 @@
...
+      - code: AcademicMajorSpecializationId
+        description: The foreign key to the Academic major specialization to which this relationship belongs.
+        fk: Academic major specialization.SisIntId
+        name: SIS Internal Academic major specialization ID
```
Comment (karaboldt) [inline udp/context-data/src/sis-2.0.yaml]: Ditto^

**Comment by pelaprat** [comment]
@karaboldt – updates made.

**Pull request merged**